src/data/thunks.js

```diff
@@ -215,6 +215,63 @@ export function skipProctoringExam() {
   };
 }
 
+/*
+Converts the given value in minutes to a more human readable format
+1 -> 1 Minute
+2 -> 2 Minutes
+60 -> 1 hour
+90 -> 1 hour and 30 Minutes
+120 -> 2 hours
+*/
+function humanizedTime(timeInMinutes) {
```

Comment:
You are passing in seconds, but the function is expecting the time to be passed in in minutes. Should be easy enough to convert from seconds to minutes in the function!

Reply:
wait never mind, just saw the other function sorry!

Reply:
On another note though, I don't think this logic belongs in thunks.js